batch/batch/worker/worker.py

```diff
@@ -964,7 +965,7 @@ async def delete(self):
     #     exit_code: int
     #   }
     # }
-    async def status(self, state=None):
+    def status(self, state=None):
```

Comment:
We never call status with a state argument. Let's remove that argument while we're making these changes.